auth-web/src/util/common-util.ts

```diff
@@ -263,4 +263,11 @@ export default class CommonUtils {
       ]
     }
   }
+
+  // format number to two places - for eg 2 => 02, 10 => 10.
+  static formatNumberToTwoPlaces (value: number): string {
+    const onesPlace = Math.floor(value % 10).toString()
```

Comment:
padStart will be good 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

Reply:
Thanks refactored